pan3d/ui/coordinate_configure.py

```diff
@@ -52,65 +57,83 @@ def __init__(
                                         v_model=axis_info["index_var"],
                                         min=0,
                                         max=(
-                                            "%s?.size - 1" % coordinate_info,
+                                            f"{coordinate_info}?.size - 1",
                                             0,
                                         ),
+                                        step=(f"{coordinate_info}?.step", 0),
```

Comment:
step of 0 by default?

(might be ok, I just want to double check as from the outside it is strange)

Reply:
Thanks, changed in [817bc5e](https://github.com/Kitware/pan3d/pull/33/commits/817bc5edce5cb29684f1bf6047e58cce5cf16111)